web/src/main/kotlin/no/nav/su/se/bakover/web/routes/revurdering/FortsettEtterForhåndsvarselRoute.kt

```diff
@@ -67,11 +68,23 @@ internal fun Route.fortsettEtterForhåndsvarselRoute(
                     }
                         .flatMap {
                             revurderingService.fortsettEtterForhåndsvarsling(it)
-                                .mapLeft {
-                                    HttpStatusCode.BadRequest.errorJson(
-                                        "Klarte ikke fortsette revurdering etter forhåndsvarsling",
-                                        "klarte_ikke_fortsette_revurdering_etter_forhåndsvarsling",
-                                    )
+                                .mapLeft { fortsettEtterForhåndsvarselFeil ->
+                                    when (fortsettEtterForhåndsvarselFeil) {
+                                        FortsettEtterForhåndsvarselFeil.AlleredeBesluttet -> HttpStatusCode.BadRequest.errorJson(
+                                            "Forhåndsvarslingen er allerede besluttet",
+                                            "forhåndsvarslingen_er_allerede_besluttet",
+                                        )
+                                        is FortsettEtterForhåndsvarselFeil.Attestering -> fortsettEtterForhåndsvarselFeil.subError.tilResultat()
+                                        FortsettEtterForhåndsvarselFeil.FantIkkeRevurdering -> Revurderingsfeilresponser.fantIkkeRevurdering
+                                        FortsettEtterForhåndsvarselFeil.RevurderingErIkkeForhåndsvarslet -> HttpStatusCode.BadRequest.errorJson(
+                                            "Revurderingen er ikke forhåndsvarslet for å kunne beslutte",
+                                            "ikke_forhåndsvarslet_for_å_kunne_beslutte",
```

Comment:
Leser litt rart, kanskje bare droppe delen om "kunne beslutte"?

```suggestion
                                            "Revurderingen er ikke forhåndsvarslet",
                                            "ikke_forhåndsvarslet",
```